### PR TITLE
Update information about xproj migration

### PR DESCRIPTION
**Pull request by drewnoakes**
`xproj` to `csproj` migration capabilites are being removed from VS2019 16.3 and CLI 3.0.100. This commit updates the documentation to reflect that.

Also some minor formatting for consistency.

Relates to dotnet/project-system#4895 and https://github.com/MicrosoftDocs/visualstudio-docs/pull/3379.

//cc: @livarcocc @KathleenDollard 

---

Created as a draft as this change refers to unreleased product versions.

### CODE REVIEW

**Comment by Thraka** [comment]
In general, we don't review PRs that are in draft mode. Please click on the **Ready for review** button when you want this reviewed.

File: docs/core/migration/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Files that were migrated (*project.json*, *global.json*, *.xproj* and solution f
 A file named *UpgradeLog.htm* is also saved and automatically opened that contains a migration report.
 
 > [!IMPORTANT]
-> The new tooling is not available in Visual Studio 2015, so you cannot migrate your projects using that version of Visual Studio.
+> The above tooling is only available between Visual Studio 2017 (version 15.0) and Visual Studio 2019 (version 16.3). If you are using a version of Visual Studio outside of that range, you should either install a suitable version of Visual Studio or use the command line migration tool described below.
```
Comment (livarcocc) [inline docs/core/migration/index.md]: It won't be in 16.3 right? So, this should say 16.2?

File: docs/core/migration/index.md [hunk 2]
```diff
@@ -46,17 +46,17 @@ When you migrate a project, the project and all its dependencies are migrated.
...
 
-### Common issues
+If you get an error:
 
-- If you get an error: "No executable found matching command dotnet-migrate":
+> No executable found matching command dotnet-migrate
 
-Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) requires .NET Core CLI RC3 or higher.
-You’ll get this error if you have a *global.json* file in the current or parent directory and the `sdk` version is set to an older version.
+Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) is only available between .NET Core CLI RC3 and 3.0.100.
```
Comment (livarcocc) [inline docs/core/migration/index.md]: It won't be in 3.0.100. That's where we pulled it out.

File: docs/core/migration/index.md (hunk 1 above)
Comment (drewnoakes) [reply docs/core/migration/index.md]: Thanks, fixed.

File: docs/core/migration/index.md (hunk 2 above)
Comment (drewnoakes) [reply docs/core/migration/index.md]: Fixed. Thanks again.

**Comment by drewnoakes** [comment]
@Thraka thanks, I'll set this as ready for review once 16.3 is released.

File: docs/core/migration/index.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ If you get an error:
 
 > No executable found matching command dotnet-migrate
 
-Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) is only available between .NET Core CLI RC3 and 3.0.100.
+Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) was introduced in .NET Core CLI RC3 and removed in version 3.0.100.
```
Comment (Thraka) [inline docs/core/migration/index.md]: RC3? What is RC3? If it's 1.0/1.1 and a release candidate, I would just state 1.0 (or whatever) and omit the preview/rc stuff.

Comment (drewnoakes) [reply docs/core/migration/index.md]: Fair point. @livarcocc can you suggest better text here?

Comment (livarcocc) [reply docs/core/migration/index.md]: It was introduced in .NET Core SDK 1.0.0? I don't think we need to call out exactly the preview version where it first came out.

Comment (drewnoakes) [reply docs/core/migration/index.md]: Thanks, updated.

**Comment by mairaw** [comment]
@drewnoakes should we resume the work on this PR? 16.3 is now released. 

Also, there are some merge conflicts now.

**Comment by drewnoakes** [comment]
@mariaw thank you for following up. I've fixed the merge conflicts. Let me know if you'd like to see anything else here. I'm happy for this to be merged if you are.

File: docs/core/migration/index.md [hunk 4]
```diff
@@ -43,17 +43,17 @@ In the command-line scenario, you can use the [`dotnet migrate`](../tools/dotnet
...
-### Common issues
+If you get an error:
 
-- If you get an error: "No executable found matching command dotnet-migrate":
+> No executable found matching command dotnet-migrate
 
-Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) requires .NET Core CLI RC3 or higher.
-You’ll get this error if you have a *global.json* file in the current or parent directory and the `sdk` version is set to an older version.
+Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) was introduced in .NET Core SDK 1.0.0 and removed in version 3.0.100.
+You'll get this error if you have a *global.json* file in the current or parent directory and the `sdk` version is set to a version outside this range.
```
Comment (gewarren) [inline docs/core/migration/index.md]:
```suggestion
You'll get this error if you have a *global.json* file in the current or parent directory, and the `sdk` version it specifies is outside this range.
```

File: docs/core/migration/index.md [hunk 5]
```diff
@@ -43,17 +43,17 @@ In the command-line scenario, you can use the [`dotnet migrate`](../tools/dotnet
 Files that were migrated (*project.json*, *global.json*, and *.xproj*) are moved to a *backup* folder.
 
 > [!NOTE]
-> If you are using Visual Studio Code, the `dotnet migrate` command does not modify Visual Studio Code-specific files such as `tasks.json`. These files need to be changed manually.
-> This is also true if you are using Project Ryder or any editor or Integrated Development Environment (IDE) other than Visual Studio.
+> If you are using Visual Studio Code, the `dotnet migrate` command will not modify Visual Studio Code-specific files such as *tasks.json*. These files need to be changed manually.
```
Comment (gewarren) [inline docs/core/migration/index.md]:
```suggestion
> If you are using Visual Studio Code, the `dotnet migrate` command does not modify Visual Studio Code-specific files such as *tasks.json*. These files need to be changed manually.
```

The MS style guide says to use present tense.

File: docs/core/migration/index.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ Visual Studio migrates the selected projects automatically. When migrating a sol
 Files that were migrated (*project.json*, *global.json*, *.xproj*, and solution file) are moved to a *Backup* folder. The migrated solution file is upgraded to Visual Studio 2017 or Visual Studio 2019 and you won't be able to open that solution file in Visual Studio 2015 or earlier versions. A file named *UpgradeLog.htm* that contains a migration report is also saved and opened automatically.
 
 > [!IMPORTANT]
-> You cannot migrate your projects using Visual Studio 2015.
+> The above tooling is only available between Visual Studio 2017 (version 15.0) and Visual Studio 2019 (version 16.2). If you are using a version of Visual Studio outside of that range, you should either install a suitable version of Visual Studio or use the command line migration tool described below.
```
Comment (gewarren) [inline docs/core/migration/index.md]:
```suggestion
> In Visual Studio 2019 version 16.3 and later, you cannot load or migrate an *.xproj* file. Additionally, Visual Studio 2015 doesn't provide the ability to migrate an *.xproj* file. If you're using one of these Visual Studio versions, either install a suitable version of Visual Studio, or use the command line migration tool that's described next.
```

**Review by gewarren — COMMENTED** [review]
Left a few suggestions for clarity.

**Comment by drewnoakes** [comment]
Thanks @gewarren, those are all nice improvements 👍

**Review by gewarren — APPROVED** [review]

**Pull request merged**